example/src/example.erl

```diff
@@ -0,0 +1,64 @@
+-module(example).
+
+-export([start/0]).
+-export([start/2]).
+-export([stop/0]).
+-export([stop/1]).
+-export([start_phase/3]).
+
+%% application
+%% @doc Starts the application
+start() ->
+ % {ok, _Started}
+  Resp = application:ensure_all_started(example).
+
+%% @doc Stops the application
+stop() ->
+  application:stop(example).
+
+%% behaviour
+%% @private
+start(_StartType, _StartArgs) ->
+  {ok, Pid} = example_sup:start_link(),
```

Comment:
no need for pattern matching here, just return what `example_sup:start_link()` returns